test/src/test_getrf.cpp

```diff
@@ -0,0 +1,97 @@
+/// @file test_getrf.cpp
+/// @brief Test the LU factorization of a matrix A
+//
+// Copyright (c) 2022, University of Colorado Denver. All rights reserved.
+//
+// This file is part of <T>LAPACK.
+// <T>LAPACK is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/generators/catch_generators.hpp>
+#include <tlapack/plugins/stdvector.hpp>
+#include <tlapack/plugins/legacyArray.hpp>
+#include <tlapack/lapack/getrf.hpp>
```

Comment:
Some tests are failing because we do not have the wrappers to BLAS in `swap.hpp`. The wrappers are on `wrappers.hpp`. Maybe each wrapper should be in one file.

For now, I suggest you replace `#include <tlapack/lapack/getrf.hpp>` by `#include <tlapack/tlapack.hpp>` as you originally wrote.

Reply:
I implemented the fix